packages/backend/dockerfile

```diff
@@ -0,0 +1,26 @@
+# Use an official Node.js image based on Alpine Linux
+FROM node:18-alpine
+
+# Set the working directory in the container
+WORKDIR /
+
+# Copy package.json and package-lock.json files
+COPY package*.json ./
+
+# Install dependencies
+RUN npm install
+
+# Install nodemon globally
+RUN npm install -g nodemon
+
+# Copy the rest of the application code to the working directory
+COPY . .
+
+# Copy the environment variables file
+COPY .env .env
+
+# Expose the port that the app runs on
+EXPOSE 3000
+
+# Command to run the application with nodemon
+CMD ["nodemon", "index.js"]
```

Comment:
Do you think you want to change this to CMD ["npm", "run", "dev"]
Using npm run dev:

You can use CMD ["npm", "run", "dev"] if you want to ensure all environment variables and other setups defined in your package.json are used.
This approach is more flexible and follows the convention of running npm scripts, which might be preferable.